src/components/Navbar.js

```diff
@@ -34,9 +34,7 @@ class Navbar extends React.Component {
 
 const styles = StyleSheet.create({
   navbar: {
-    position: 'absolute', 
-    height: 85, 
-    bottom: 0, 
+    height: 85,
     width: '100%', 
     flexDirection: 'row', 
     backgroundColor: colors.white,
```

Comment:
So we didn't need position or bottom on this to stay put down below? Is that because this is now controlled in the ComponentWithNavbar.js?

Reply:
right, in ComponentWithNavbar the navbar is a set height and the scroll view is flex so it will take up all remaining space. I tested on several pages